api/src/main/java/com/epam/pipeline/controller/metadata/CategoricalAttributeController.java

```diff
@@ -0,0 +1,126 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.epam.pipeline.controller.metadata;
+
+import com.epam.pipeline.controller.AbstractRestController;
+import com.epam.pipeline.controller.Result;
+import com.epam.pipeline.manager.metadata.CategoricalAttributeApiService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.List;
+import java.util.Map;
+
+@Controller
+@Api(value = "CategoricalAttributes")
+@RequiredArgsConstructor
+@RequestMapping(value = "/categoricalAttribute")
+public class CategoricalAttributeController extends AbstractRestController {
+
+    private final CategoricalAttributeApiService categoricalAttributeApiService;
+
+    @PostMapping
+    @ResponseBody
+    @ApiOperation(
+        value = "Add categorical attributes values.",
+        notes = "Add categorical attributes values",
+        produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiResponses(
+        value = {@ApiResponse(code = HTTP_STATUS_OK, message = API_STATUS_DESCRIPTION)
+        })
+    public Result<Boolean> insertCategoricalAttributes(@RequestBody final Map<String, List<String>> dict) {
+        return Result.success(categoricalAttributeApiService.insertAttributesValues(dict));
+    }
+
+    @GetMapping("/loadAll")
+    @ResponseBody
+    @ApiOperation(
+        value = "Load all categorical attributes with values.",
+        notes = "Load all categorical attributes with values.",
+        produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiResponses(
+        value = {@ApiResponse(code = HTTP_STATUS_OK, message = API_STATUS_DESCRIPTION)
+        })
+    public Result<Map<String, List<String>>> loadAllCategoricalAttributes() {
+        return Result.success(categoricalAttributeApiService.loadAll());
+    }
+
+    @GetMapping("/load")
+    @ResponseBody
+    @ApiOperation(
+        value = "Load all requested attributes with values.",
+        notes = "Load all requested attributes with values.",
+        produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiResponses(
+        value = {@ApiResponse(code = HTTP_STATUS_OK, message = API_STATUS_DESCRIPTION)
+        })
+    public Result<Map<String, List<String>>> loadCategoricalAttributes(@RequestParam final List<String> attributeKeys) {
```

Comment:
I'd suggest to allow only one `attributeKey` for this method since we already have a method to loadAll. And let's move to path from request params

Reply:
Done, extracted to the path